global/scripts/MyelinMap_BC.sh

```diff
@@ -156,13 +157,19 @@ ${Caret7_Command} -cifti-resample ${LowResBiasField} \
 log_Msg "Resampled BiasField in the native mesh space: ${NativeBiasField}"
 
 # generate bias corrected map in the output mesh space
-NativeBCMap=${NativeFolder}/${Subject}.MyelinMap_BC${RegNameInOutputName}.native.dscalar.nii
+NativeBCMapToUse=${NativeFolder}/${Subject}.MyelinMap_BC${RegNameInOutputName}.native.dscalar.nii
+NativeMyelinMapToUse=${NativeMyelinMap}
+if [[ "$MapName" != "MyelinMap" ]]; then
```

Comment:
It looks like when MapName is MyelinMap, the lines inside this `if` would result in the same filenames, so this conditional should be unnecessary.  Please try to simplify the logic and remove any variables that aren't needed.

Reply:
fixed